src/hooks/Form.tsx

```diff
@@ -23,13 +23,31 @@ export const STATISTICS_QUERY_KEY = 'statistics';
 
 export const useFormById = (formId: string) =>
   useQuery<Form, RequestResponse>([FORM_QUERY_KEY, formId], () => API.getForm(formId), { enabled: formId !== '-' });
+export const useFormTemplates = () => useQuery<Array<Form>, RequestResponse>([FORM_QUERY_KEY, 'templates'], () => API.getFormTemplates(), {});
+
+export const useDeleteFormTemplate = (formId: string): UseMutationResult<RequestResponse, RequestResponse, undefined, unknown> => {
```

Comment:
Trengs det en egen hook for dette eller er det mulig å gjenbruke den eksisterende `useDeleteForm`